pkg/database/realm.go

```diff
@@ -170,6 +169,13 @@ type Realm struct {
 	LongCodeLength   uint            `gorm:"type:smallint; not null; default: 16;"`
 	LongCodeDuration DurationSeconds `gorm:"type:bigint; not null; default: 86400;"` // default 24h
 
+	// ShortCodeMaxMinutes can only be set by system admins and allows for a
+	// realm to have a higher max short code duration
+	ShortCodeMaxMinutes uint
```

Comment:
Unfortunately, since realm is already managed by gorm, you need the gorm annotations here. A drop and re-recreate of the database will create these fields because of AutoMigrate.

Reply:
done.